### ISSUE TITLE
Extent button does not respond

### ISSUE THREAD

**Issue by Charence** [3drepo/3drepo.io#556]
The extent button does not respond to mouse input.

Console says the following:
```
TypeError: e.goToExtent is not a function
    at Object.t.extent [as click] (bottom-buttons.component.js:71)
    at fn (eval at compile (three_d_repo.dependencies.min.js:18868), <anonymous>:4:173)
    at r (three_d_repo.dependencies.min.js:30653)
    at p.$eval (three_d_repo.dependencies.min.js:21740)
    at p.$apply (three_d_repo.dependencies.min.js:21840)
    at HTMLSpanElement.<anonymous> (three_d_repo.dependencies.min.js:30658)
    at Ze (three_d_repo.dependencies.min.js:7139)
    at HTMLSpanElement.n (three_d_repo.dependencies.min.js:7127)
(anonymous) @ three_d_repo.dependencies.min.js:18010
```

Checked versions: 0eea6080df7739adbe7acc9dea9c1ce525be37d8 and dev

**Comment by JamesMilnerUK** [comment]
My bad, good spot. I'll sort this, simple typo on the name.

**Comment by JamesMilnerUK** [comment]
Fixed in #545 . Also fixed a bug in unity util that was stoping you from going inside an issue whilst loading, or loading an issue direct from a URL

**Comment by JamesMilnerUK** [comment]
Turns out  I did not fix this :P But it is fixed now on the associated branch, merged into staging
